wandb/integration/keras/keras.py

```diff
@@ -698,11 +700,14 @@ def on_train_begin(self, logs=None):
                     f"Skipping logging validation data. Error: {str(e)}"
                 )
 
-        if _can_compute_flops():
+        if self.compute_flops and _can_compute_flops():
             try:
                 wandb.summary["GFLOPs"] = self.get_flops()
             except Exception as e:
-                wandb.termwarn("Unable to compute FLOPs for this model.")
```

Comment:
I would suggest to simply add `repeat=False` here so that the term warning is only printed once.

Reply:
Hey @dmitryduev, the `get_flops` method is called just once `on_train_begin` thus, adding `repeat=False ` will not be helpful IMO. As per the discussion [here](https://wandb.atlassian.net/browse/WB-10339), we would like to fail it silently and mention it in the documentation.

Reply:
How about this report: https://github.com/wandb/wandb/issues/3916

Reply:
cc: @morganmcg1 what do you think?

Reply:
@ayulockin I think `repeat=False` makes sense

Reply:
@ayulockin any updates? do you want this in the upcoming release?